api-docs/controller/paths.adoc

```diff
@@ -862,15 +862,17 @@ POST /controller/management/servers/{serverTemplateId}/containers/{containerId}
 
 ===== Parameters
 
-[options="header", cols=".^2a,.^3a,.^9a,.^4a"]
+[options="header", cols=".^2a,.^3a,.^9a,.^4a,.^2a"]
 |===
-|Type|Name|Description|Schema
+|Type|Name|Description|Schema|Default
 |**Path**|**containerId** +
-__required__|ID of the KIE container to be updated|string
+__required__|ID of the KIE container to be updated|string|
 |**Path**|**serverTemplateId** +
-__required__|ID of the KIE Server template associated with the KIE container|string
+__required__|ID of the KIE Server template associated with the KIE container|string|
+|**Query**|**resetBeforeUpdate** +
+__optional__|Allows to reset the current environment aborting active process instances before updating when the server runs on development mode. Optional, defaults to false|boolean|`"false"`
```

Comment:
Suggest this for consistency:

``
Determines whether active processes are aborted (reset) before updating when the server runs in development mode (default: `false`)
```

Reply:
@sterobin I created another PR to handle this update, see https://github.com/kiegroup/droolsjbpm-integration/pull/1778
If you don't mind, I prefer to merge this PR as is and get that change in here using a separated PR.

Reply:
Okay, go ahead and merge then.